src/authorization_service_configuration.ts

```diff
@@ -70,7 +70,22 @@ export class AuthorizationServiceConfiguration {
 
   static fetchFromIssuer(openIdIssuerUrl: string, requestor?: Requestor):
       Promise<AuthorizationServiceConfiguration> {
-    const fullUrl = `${openIdIssuerUrl}/${WELL_KNOWN_PATH}/${OPENID_CONFIGURATION}`;
+    const searchForQueryParams = function(url: string) {
```

Comment:
Not entirely sure what platform-agnostic means given that the URL assumes it would be a valid one but hope the new changes suffice the requested change